vaadin-grid-flow-parent/vaadin-grid-testbench/src/main/java/com/vaadin/flow/component/grid/testbench/GridTRElement.java

```diff
@@ -30,13 +35,33 @@ public class GridTRElement extends TestBenchElement {
      * @return the cell for the given column
      */
     public GridTHTDElement getCell(GridColumnElement column) {
-        TestBenchElement e = (TestBenchElement) executeScript(
```

Comment:
This change revealed another bug in the GridElement helper (some unrelated tests were failing). Pushed a fix in https://github.com/vaadin/flow-components/pull/5508/commits/e9d84c2ffa9cc916705d11b73fb89baae1c6b0d8